job/helpers/scrape_config.py

```diff
@@ -0,0 +1,10 @@
+from typing import TypedDict
```

Comment:
⛏ Just to stay consistent with our naming conventions for helper modules (see `datetime.py` and `itertools.py`), I'd name this file `requests.py`.

Reply:
Great. I will do that!